### ISSUE TITLE
[Enhancement] Improve activity log details for project phase changes

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-data-tech#13218]
If a user changes a project's current phase, we see up to two events in the activity log:
1. One event for the phase that was marked as current = false
2. One event for the phase that was marked as current = true

<img width="1200" alt="Screenshot 2023-08-03 at 1 08 38 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/14793120/6764e698-389f-4857-9833-8fdb184d58c0">

The activity log should only show one entry when the current phase changes:
> Changed the current phase from (old phase name) to (new phase name)

For discussion / consideration / TBD:
- How to handle when a phase has multiple edits, including the current phase marker?
- How to handle when the current phase is removed on one phase and no other phase is set to current?

**Comment by chiaberry** [comment]
this feels bigger than a snackoo to me, since we are going to have to figure out how activity log entries will have knowledge of other entries and which ones match up

**Comment by johnclary** [comment]
you're right. we can scrap this idea. instead i would just like it to be clear when the phase is being marked current or not.
